src/js/components/Text/Text.js

```diff
@@ -91,20 +91,22 @@ const Text = forwardRef(
       </StyledText>
     );
 
+    const tipProps = tipProp && typeof tipProp === 'object' ? tipProp : {};
```

Comment:
This seems odd to me. If tipProp is defined but not an object why would we do {}? It seems like that loses tipProp.

Reply:
tipProp is either a string (which will be used for the content of the tooltip) or an object. If tipProp is a string then it doesn't make sense to pass that string as a prop to the Tooltip component. It should instead be passed as the child or through the content prop. This is why I set tipProps to an empty object if it is not an object.

Reply:
Oh interesting, good callout